airflow/utils/json.py

```diff
@@ -40,6 +43,16 @@
 
 log = logging.getLogger(__name__)
 
+CLASSNAME = "__classname__"
+VERSION = "__version__"
+DATA = "__data__"
```

Comment:
Rather than creating yet another way of encoding this kind of thing in the code base, is it possible we could reuse the `__var` and `__type` keys/prefixes that the BaseSerialization uses, or would that cause problems?

Reply:
as mentioned `__type` is overloaded and as the deserializer receives the nested dict first it is hard to distinguish between actual type and encoding type.  Futhermore, I do think __var and __type are misnomers `__type` is not pythonic and `__var` is incorrect imho (it is the data of an object, not the a variable).